judgments/tests/utils/assertions.py

```diff
@@ -2,22 +2,55 @@
 Custom assertion helpers for tests.
 """
 
+from lxml import html
 
-def assert_contains_html(response, html):
-    """
-    Asserts that the given HTML is contained within the response content.
 
-    Raises:
-        AssertionError: If the HTML is not found in the response content.
-    """
-    assert html.replace(" ", "").replace("\n", "") in response.content.decode().replace(" ", "").replace("\n", "")
+def parse_html(html_string):
+    return html.fromstring(html_string)
 
 
-def assert_not_contains_html(response, html):
-    """
-    Asserts that the given HTML is not contained within the response content.
+def parse_html_fragment(html_string):
+    return html.fragments_fromstring(html_string)
 
-    Raises:
-        AssertionError: If the HTML is found in the response content.
-    """
-    assert html.replace(" ", "").replace("\n", "") not in response.content.decode().replace(" ", "").replace("\n", "")
+
+def normalise_text(text):
+    return " ".join(text.split())
+
+
+def elements_match(element_1, element_2):
```

Comment:
This feels complicated and could do with some tests. Yes this is starting to feel recursive :D